packages/kuma-gui/src/app/kuma/components/ApplicationShell.vue

```diff
@@ -5,179 +5,230 @@
     <XTeleportSlot
       name="modal-layer"
     />
-    <header
-      role="banner"
+    <DataSource
+      :src="`/me/~notifications`"
+      v-slot="{ data: dismissed, refresh }"
     >
-      <div
-        class="horizontal-list"
+      <XNotificationHub
+        v-if="dismissed"
+        uri="application-shell"
+        :dismissed="dismissed"
+        v-slot="hub"
       >
-        <slot name="header">
-          <XAction :to="{ name: 'home' }">
-            <slot name="home" />
-          </XAction>
-
-          <GithubButton
-            class="gh-star"
-            href="https://github.com/kumahq/kuma"
-            aria-label="Star kumahq/kuma on GitHub"
+        <XNotification
+          v-if="!can('use state')"
+          uri="global-store-type-memory"
+        >
+          <XI18n
+            path="common.warnings.GLOBAL_STORE_TYPE_MEMORY"
+          />
+        </XNotification>
```

Comment:
I plan on moving this notification out of ApplicationShell and into an AppView once https://github.com/kumahq/kuma-gui/issues/3570 is done. but this shouldn't block approval/merge.